modules/prometheus-adapter/README.md

```diff
@@ -0,0 +1,20 @@
+This module deploys a [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) and a [Prometheus frontend](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/examples/frontend.yaml) to a cluster. See [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) repo for more details.
+
+## Bash equivalent of this module
+
+Assure the following are set before running:
+   - PROJECT_ID: GKE Project ID
+   - (optional) PROMETHEUS_HELM_VALUES_FILE: Values file to pass when deploying `prometheus-community/prometheus-adapter` chart
+
+```
+curl https://raw.githubusercontent.com/GoogleCloudPlatform/prometheus-engine/v0.10.0/examples/frontend.yaml | envsubst
+
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+
+if [ -z "$PROMETHEUS_HELM_VALUES_FILE" ]
+    helm install jetstream prometheus-community/prometheus-adapter
```

Comment:
Does this work for non jetstream workloads?

If so, we should parameterize the name "jetstream" here